app/subscribers/foreman_rh_cloud/insights_subscriber.rb

```diff
@@ -0,0 +1,9 @@
+module ForemanRhCloud
+  class InsightsSubscriber < ::Foreman::BaseSubscriber
+    def call(*args)
+      host = args.first.payload[:object]
+      host_status = host.get_status(InsightsClientReportStatus)
+      host_status.update(status: host_status.to_status)
```

Comment:
I don't mind checking in as it is, but there is `host.refresh_statuses([InsightsClientReportStatus])` method that would do the same.

Reply:
I'd like to keep it as it is because `refresh_statuses` also refresh timestamps which is something I don't want in this step.